stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/channel/internal/DatabaseChannelRepository.kt

```diff
@@ -37,23 +36,18 @@ internal class DatabaseChannelRepository(
     private val channelDao: ChannelDao,
     private val getUser: suspend (userId: String) -> User,
     private val getMessage: suspend (messageId: String) -> Message?,
-    cacheSize: Int = 100,
 ) : ChannelRepository {
 
     private val logger by taggedLogger("Chat:ChannelRepository")
 
-    // the channel cache is simple, just keeps the last several users in memory
-    private val channelCache = LruCache<String, Channel>(cacheSize)
-
     /**
      * Inserts a [Channel]
      *
      * @param channel [Channel]
      */
-    override suspend fun insertChannel(channel: Channel) {
-        val entity = channel.toEntity()
+    override suspend fun upsertChannel(channel: Channel) {
+        val entity = channel.parseEntity()
         logger.v { "[insertChannel] entity: ${entity.lastMessageInfo()}" }
```

Comment:
Could we alter logs in respect of new func names?
```suggestion
        logger.v { "[upsertChannel] entity: ${entity.lastMessageInfo()}" }
```